app/presenters/two_factor_auth_code/webauthn_authentication_presenter.rb

```diff
@@ -5,23 +5,58 @@ class WebauthnAuthenticationPresenter < TwoFactorAuthCode::GenericDeliveryPresen
 
     attr_reader :credential_ids, :user_opted_remember_device_cookie
 
+    def initialize(data:, view:, remember_device_default: true, platform_authenticator: true)
+      @platform_authenticator = true
```

Comment:
should this use the param?
```suggestion
    def initialize(data:, view:, remember_device_default: true, platform_authenticator: true)
      @platform_authenticator = platform_authenticator
```

Reply:
nice, yes we should

Reply:
I needed this change for testing 8f1b997, so I patched it in in 4ce3b97.

Reply:
@aduth confirming. this is resolved, correct?

Reply:
> @aduth confirming. this is resolved, correct?

It is, yes 👍